twitter/twitter.php

```diff
@@ -173,6 +174,39 @@ function twitter_follow(App $a, array &$contact)
 	}
 }
 
+function twitter_unfollow(App $a, array &$hook_data)
+{
+	$contact = $hook_data['contact'];
+	Logger::info('Check if contact is twitter contact', ['url' => $contact["url"]]);
+
+	if (!strstr($contact["url"], "://twitter.com") && !strstr($contact["url"], "@twitter.com")) {
```

Comment:
Wouldn't it be better to simply check if the `network` was set to `Protocol:TWITTER`?

Reply:
Yes, I copied this code from an earlier function about following when we don't have this info yet.